src/modules/i18n/locales/de.json

```diff
@@ -484,6 +484,10 @@
     "time_hide": "Deaktivieren der Anzeige von Daten-Zeitständen.",
     "time_select_year": "Wählen Sie ein Jahr aus",
     "time_show": "Aktivieren der Anzeige von Daten-Zeitständen.",
+    "time_slider_legend_tippy_intro": "Der Zeitschieber zeigt verschiedene Farben an, je nachdem, ob die Zeitschichten Daten für das ausgewählte Jahr haben oder nicht.",
```

Comment:
"haben" durch "enthalten" ersetzten ?

Reply:
The word "Zeitschichten" is strange. I'd use "ob die dargestellten Layer Daten für das ausgewählte ..."
We use "Layer" in other places for the German texts, and that seems OK to me.

Reply:
OK, just saw that for EN we use map instead of layer, so it would be "ob die dargestellten Karten für das ..."